shanoir-ng-front/src/app/shared/localLanguage/localDateFormat.pipe.ts

```diff
@@ -0,0 +1,23 @@
+import {Pipe, PipeTransform} from "@angular/core";
+import {BROWSER_LANGUAGE, dateFormat} from "./localDate.abstract";
+import { DatePipe } from '@angular/common';
+
+@Pipe({
+    name: 'localDateFormatPipe',
+})
+
+export class LocalDateFormatPipe implements PipeTransform{
+
+    private datePipe = new DatePipe(BROWSER_LANGUAGE);
+
+    transform(date: Date, time: Boolean = false) {
```

Comment:
not a big deal but it is better to use the primitive 'boolean' instead of the class 'Boolean'